pkg/image/image_test.go

```diff
@@ -97,13 +98,17 @@ func TestImageAdditionalMetadata(t *testing.T) {
 				os.Remove(tempFile.Name())
 			})
 
-			img := NewImage(nil, tempFile.Name(), test.options...)
+			img := New(nil, tempFile.Name(), test.options...)
 
 			err = img.applyOverrideMetadata()
 			if err != nil {
 				t.Fatalf("could not create image: %+v", err)
 			}
-			for _, d := range deep.Equal(img, &test.image) {
+			if d := cmp.Diff(img, &test.image,
```

Comment:
nit: could an `assert.JSONEq` work here? The output can be easier to read than the `cmp.Diff`

Reply:
Since this is with struct instances and am not planning in testing the json representation of these I think something like cmp.Diff is the right tool here.